src/components/Popup/BackupPage.vue

```diff
@@ -17,23 +17,47 @@
       <a-button-link
         download="authenticator.txt"
         :href="exportOneLineOtpAuthFile"
-        v-if="!unsupportedAccounts"
+        v-if="!unsupportedAccounts & isDataLinkSupported"
         >{{ i18n.download_backup }}</a-button-link
       >
-      <a-button-link download="authenticator.json" :href="exportFile" v-else>{{
-        i18n.download_backup
-      }}</a-button-link>
+      <button
+        v-on:click="downloadBackUpOneLineOtpAuthFile()"
+        v-if="!unsupportedAccounts & !isDataLinkSupported"
```

Comment:
Shouldn't this be `&&`? `&` is a bitwise and. This applies to the whole file.

Reply:
Thanks good catch!